tests/x11/toolkits/prepare.pm

```diff
@@ -26,6 +26,8 @@ sub run {
             assert_script_run 'SUSEConnect -p sle-module-development-tools/${VERSION_ID}/${CPU}';
         }
         else {
+            # for some reason we don't register SLE12 systems in openqa by default
+            assert_script_run 'SUSEConnect -r ' . get_var('SCC_REGCODE');
```

Comment:
maybe you can do it in single call ? SUSEConnect allow it

Reply:
There is also lib/suseconnect_register.pm with some helpful functions